### PR TITLE
refactor(three_nodes test, .circleci/config.yml): fixed account usernames to be clearer, etc

### PR DESCRIPTION
**Pull request by dora-gt**
To investigate more about #490, fixed some lines to output more detailed information.
I could not reproduce errors on my computer, so I need more information in the CI process.

### CODE REVIEW

File: crates/ilp-node/tests/three_nodes.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
-use futures::{future::join_all, Future};
+use futures::{future::join_all, stream::*, sync::mpsc, Future};
 use ilp_node::InterledgerNode;
+use log::debug;
```
Comment (emschwartz) [inline crates/ilp-node/tests/three_nodes.rs]: You should just use `tracing::debug` instead. It's functionality is a superset of `log`'s

File: crates/ilp-node/tests/three_nodes.rs [hunk 2]
```diff
@@ -42,35 +45,35 @@ fn three_nodes() {
 
     let alice_on_alice = json!({
         "ilp_address": "example.alice",
-        "username": "alice",
+        "username": "alice_on_alice",
```
Comment (emschwartz) [inline crates/ilp-node/tests/three_nodes.rs]: I think it's even clearer if the nodes are identified as `a` and `b` so it's `alice_on_a`, etc

File: crates/ilp-node/tests/three_nodes.rs [hunk 3]
```diff
@@ -137,16 +139,24 @@ fn three_nodes() {
...
 
+    let mut node1_finish_sender = finish_sender.clone();
     runtime.spawn(
         node1
             .serve()
             .and_then(move |_| alice_fut)
-            .and_then(|_| Ok(()))
+            .and_then(move |_| {
+                node1_finish_sender
+                    .try_send(1)
```
Comment (emschwartz) [inline crates/ilp-node/tests/three_nodes.rs]: What is this for?

File: crates/ilp-node/tests/three_nodes.rs (hunk 2 above)
Comment (dora-gt) [reply crates/ilp-node/tests/three_nodes.rs]: Indeed. I'll fix.

File: crates/ilp-node/tests/three_nodes.rs (hunk 3 above)
Comment (dora-gt) [reply crates/ilp-node/tests/three_nodes.rs]: I'm not sure this has positive effects but it just ensures that tasks of `node3` starts after `node1` and `node2` finished their tasks. It seemed more robust than waiting `1000ms` or something though it looks actually almost the same.

File: crates/ilp-node/tests/three_nodes.rs (hunk 1 above)
Comment (dora-gt) [reply crates/ilp-node/tests/three_nodes.rs]: Fixed.

File: crates/ilp-node/tests/three_nodes.rs (hunk 2 above)
Comment (dora-gt) [reply crates/ilp-node/tests/three_nodes.rs]: Fixed.

**Comment by dora-gt** [comment]
I'm merging this.
I'll check logs if the tests fail again.

**Pull request merged**